### ISSUE TITLE
Expand Anvil Color patch to support HEX & disabling forced italic.

### ISSUE THREAD

**Issue by mibby** [pl3xgaming/Purpur#153]
Would it be possible to expand the Anvil GUI coloring to support HEX codes and an option to disable the forced always-on italic? A fork of the plugin ColoredAnvils supports this https://github.com/Kikisito/ColoredAnvils but it would be nice to have it integrated into purpur's own patch.

**Comment by BillyGalbreath** [comment]
hex in anvils should already be possible using md_5's ugly `&x` color codes. paper is in the works on integrating the adventure and minimessage apis. once this is done a lot of these old patches of mine will be updated to use those apis.

as for the forced italic, that is actually a bug in spigot. it will be also solved when adventure is integrated.

**Comment by mibby** [comment]
Do you happen to know of any timeframe when the apis would be merged in Paper?
